### PR TITLE
chore: make localnet external-dev friendly 😇

### PR DESCRIPTION
**Pull request by ahasna**
No longer require docker login for running localnets.

### CODE REVIEW

**Review by kylezs — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -30,7 +30,25 @@ cargo cf-test-ci
 > **_NOTE:_**  cf-test-ci is an alias for cargo test with additional flags. These aliases are defined in [.cargo/config.toml](.cargo/config.toml).
 
 ## Contributing
+### Setup
```
Comment (dandanlen) [inline README.md:33]:
```suggestion

### Setup
```

File: README.md [hunk 2]
```diff
@@ -30,7 +30,25 @@ cargo cf-test-ci
...
+
+# Install Rust
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+
+# Install Node.js and pnpm
+curl -fsSL https://deb.nodesource.com/setup_18.x | sudo -E bash -
+sudo apt install -y nodejs
+npm install -g pnpm
+
+```
```
Comment (dandanlen) [inline README.md:51]:
````suggestion
```

````

File: README.md [hunk 3]
```diff
@@ -30,7 +30,25 @@ cargo cf-test-ci
 > **_NOTE:_**  cf-test-ci is an alias for cargo test with additional flags. These aliases are defined in [.cargo/config.toml](.cargo/config.toml).
 
 ## Contributing
+### Setup
+Make sure you have the following packages and tools installed. The following is for debian-like systems (e.g. Ubuntu). You may need to adjust for your system.
 
+```bash
```
Comment (dandanlen) [inline README.md:36]:
```suggestion
```bash copy
```
?

**Review by dandanlen — APPROVED** [review]
Some tiny things the annoying linter in my head noticed... (I can't turn it off)

**Pull request merged**